packages/document-grounding/README.md

```diff
@@ -0,0 +1,99 @@
+# @sap-ai-sdk/document-grounding
+
+SAP Cloud SDK for AI is the official Software Development Kit (SDK) for **SAP AI Core**, **SAP Generative AI Hub**, and **Orchestration Service**.
+
+This package incorporates generative AI document grounding capabilities into your AI activities in SAP AI Core and SAP AI Launchpad.
+
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
```

Comment:
[pp] I think we should remove this option from all READMEs. Seems redundant

Reply:
This was a mistake. Removed by using ###